keystore/filesystem/server_keystore.go

```diff
@@ -555,19 +576,26 @@ func (store *KeyStore) getPrivateKeyByFilename(id []byte, filename string) (*key
 	defer store.lock.Unlock()
 	encryptedKey, ok := store.cache.Get(filename)
 	if !ok {
-		encryptedPrivateKey, err := store.loadPrivateKey(store.GetPrivateKeyFilePath(filename))
+		loadKeyCallback := func() ([]byte, error) {
+			encryptedPrivateKey, err := store.loadPrivateKey(store.GetPrivateKeyFilePath(filename))
+			if err != nil {
+				return nil, err
+			}
+			return encryptedPrivateKey.Value, nil
```

Comment:
Why not to simplify? :)
```suggestion
			return store.loadPrivateKey(store.GetPrivateKeyFilePath(filename))
```

Reply:
Oops, it's impossible due to `.Value`, sorry :sweat_smile: